Userland/DevTools/HackStudio/Project.cpp

```diff
@@ -48,12 +48,7 @@ void Project::for_each_text_file(Function<void(const ProjectFile&)> callback) co
 NonnullRefPtr<ProjectFile> Project::get_file(const String& path) const
```

Comment:
Now that this method only creates new `ProjectFiles`s, let's call it `create_file`